docs/apache-airflow-providers-amazon/redirects.txt

```diff
@@ -15,8 +15,17 @@
 # specific language governing permissions and limitations
 # under the License.
 
-operators/s3_to_redshift.rst operators/transfer/s3_to_redshift.rst
-operators/google_api_to_s3_transfer.rst operators/transfer/google_api_to_s3.rst
-operators/imap_attachment_to_s3.rst operators/transfer/imap_attachment_to_s3.rst
-operators/salesforce_to_s3.rst operators/transfer/salesforce_to_s3.rst
+operators/s3_to_redshift.rst transfer/s3_to_redshift.rst
+operators/google_api_to_s3_transfer.rst transfer/google_api_to_s3.rst
+operators/imap_attachment_to_s3.rst transfer/imap_attachment_to_s3.rst
+operators/salesforce_to_s3.rst transfer/salesforce_to_s3.rst
 operators/sqs_publish.rst operators/sqs.rst
+operators/emr.rst operators/emr/emr.rst
+operators/emr_eks.rst operators/emr/emr_eks.rst
+operators/emr_serverless.rst operators/emr/emr_serverless.rst
+operators/redshift_sql.rst operators/redshift/redshift_sql.rst
+operators/redshift_data.rst operators/redshift/redshift_data.rst
+operators/redshift_cluster.rst operators/redshift/redshift_cluster.rst
+operators/s3.rst operators/s3/s3.rst
+operators/glacier.rst operators/s3/glacier.rst
+operators/transfer/*.rst transfer/*.rst
```

Comment:
this one might not work as i imagine.
i might need to change it to manually specify all rst files.
sadly there is no easy way to test it

Reply:
ok this is wrong. I have to specify the list manually.